### PR TITLE
Add Danish krone currency

### PR DESCRIPTION
**Pull request by FrederikBolding**
## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
Thanks for the contribution @FrederikBolding ! Is our bad that we didn't update properly the docs :( Assets follow now a different approach to be added into rotki and now we use this repo https://github.com/rotki/assets with sql sentences. 

Can you undo your changes in the backend? (the ones at `rotkehlchen/`) so we can add the asset in the other PR and then merge your PR with the changes in the frontend?

Sorry for the inconvenience

**Comment by LefterisJP** [comment]
We have an issue for updating the docs since July @yabirgb. I assigning you to it :rabbit:   -> https://github.com/rotki/rotki/issues/3227

**Comment by FrederikBolding** [comment]
> Thanks for the contribution @FrederikBolding ! Is our bad that we didn't update properly the docs :( Assets follow now a different approach to be added into rotki and now we use this repo https://github.com/rotki/assets with sql sentences.
> 
> Can you undo your changes in the backend? (the ones at `rotkehlchen/`) so we can add the asset in the other PR and then merge your PR with the changes in the frontend?
> 
> Sorry for the inconvenience

Oh, sorry! I have reverted those changes. Do you need me to do something in the assets repo then?

**Comment by yabirgb** [comment]
> > Thanks for the contribution @FrederikBolding ! Is our bad that we didn't update properly the docs :( Assets follow now a different approach to be added into rotki and now we use this repo https://github.com/rotki/assets with sql sentences.
> > Can you undo your changes in the backend? (the ones at `rotkehlchen/`) so we can add the asset in the other PR and then merge your PR with the changes in the frontend?
> > Sorry for the inconvenience
> 
> Oh, sorry! I have reverted those changes. Do you need me to do something in the assets repo then?

Thanks! Don't worry I'll handle it, is just one line :)

File: rotkehlchen/api/v1/encoding.py [hunk 1]
```diff
@@ -400,6 +400,39 @@ def _deserialize(
...
+            attr: Optional[str],  # pylint: disable=unused-argument
+            data: Optional[Mapping[str, Any]],  # pylint: disable=unused-argument
+            **_kwargs: Any,
+    ) -> Optional[Asset]:
+        try:
+            asset = Asset(value, form_with_incomplete_data=self.form_with_incomplete_data)
+        except DeserializationError as e:
+            raise ValidationError(str(e)) from e
+        except UnknownAsset:
+            log.error(f'Failed to deserialize asset {value}')
```
Comment (LefterisJP) [inline rotkehlchen/api/v1/encoding.py:431]: Does this solution really work? Wouldn't it just create a list of assets and `None` which would end up being fed to https://github.com/rotki/rotki/blob/c3c9f57b51d7bd1988d4d17cd6d3cc09284e9d63/rotkehlchen/api/rest.py#L406?

Ah wait I just saw your change in v1/resources.py .... hmm ... I don't really like this.

Or well at least  add some more info on the the log entry. That a non-supported asset was passed to the API.

**Review by LefterisJP — APPROVED** [review]
LGTM. Not super happy with the method, but when sat to think it through could not find a better way to add this fix. Also left a comment on the log entry but @yabirgb you can address it in any other PR. No need to re-run tests on this one.

**Comment by LefterisJP** [comment]
Also @FrederikBolding thank you for your PR :)

**Comment by LefterisJP** [comment]
Oh and in case you get tempted to try the develop branch @FrederikBolding be warned. Never us it with a production user database and back everything up before using the develop branch.

Until release time all DB upgrades and changes are non-finalized so things can break if you use develop with a production DB.

**Pull request merged**